charts/am-role-assignment-service/values.yaml

```diff
@@ -36,3 +36,4 @@ java:
     ROLE_ASSIGNMENT_IDAM_ADMIN_USERID: ras-validation-system@hmcts.net
     APPLICATION_LOGGING_LEVEL: INFO
     BYPASS_ORG_DROOL_RULE: false
+    RUN_DB_MIGRATION_ON_STARTUP: false
```

Comment:
you need to enable this in `values.preview.template.yaml` so it works in preview

Reply:
sure, I have enabled this migration only for the Local and PR.